hudi-sync/hudi-hive-sync/src/test/java/org/apache/hudi/hive/testutils/HiveTestUtil.java

```diff
@@ -126,6 +126,16 @@ public static void setUp() throws IOException, InterruptedException, HiveExcepti
     clear();
   }
 
+  public static void clearIncrementalPullSetup(String path1, String path2) throws IOException, HiveException, MetaException {
```

Comment:
are these paths that pass in `null`?  wondering if we can directly call `fs.delete()` here. and avoid the exists check as well.